docs/explanations/architecture/entities.md

```diff
@@ -42,13 +42,13 @@ Since the WordPress editors allow multiple entity records to be edited at the sa
 
 And to be able to perform both undo and redo operations propertly, each modification in the list of edits contains the following information:
 
- - Entity kind and name: Each entity in core-data is identified by a tuple kind, name. This corresponds to the identifier of the modified entity. 
+ - Entity kind and name: Each entity in core-data is identified by the pair _(kind, name)_. This corresponds to the identifier of the modified entity. 
```

Comment:
I chose "pair" to reduce confusion for developers on whether they should actually pass a tuple `[ 123, 456 ]` to our API's functions, when actually the pair is always flattened into the rest of the arguments.